build.gradle

```diff
@@ -93,11 +90,11 @@ dependencies {
         maven {
             url "http://download.crashlytics.com/maven"
```

Comment:
The Crashalytics link is no longer necessary as all it's stuff is in the Maven public repo.